extras/create_send_notifications.py

```diff
@@ -42,26 +42,41 @@ def create_output_directories():
         os.path.join(NOTIFICATIONS_BASE_DIR, NOTIFICATION_ARCHIVE_DIR)
     )
 
+def build_notifications_org_list(db):
+    """Return notifications organization list.
 
-def build_cyhy_org_list(db):
-    """Build list of CyHy organization IDs.
-
-    This is the list of CyHy organization IDs (and their descendants) that
-    receive CyHy reports.
+    This is the list of organization IDs that should
+    have a notification generated and sent.
+    """
+    notifications_to_generate = set()
+    notifications_to_delete = set(ticket_owner_ids) - notifications_to_generate
```

Comment:
This line needs to be moved to the end of this function, just before the `return` statement.  It will throw an error if you don't move this, since `ticket_owner_ids` is undefined at this point.

Reply:
@dav3r Please see https://github.com/cisagov/cyhy-reports/pull/96/commits/6bc09be9bbf4bb441a65d31efbee6f659bbaafca

Reply:
You moved the wrong line- you should revert https://github.com/cisagov/cyhy-reports/commit/6bc09be9bbf4bb441a65d31efbee6f659bbaafca and instead move this line to the end:
```python
notifications_to_delete = set(ticket_owner_ids) - notifications_to_generate
```

Reply:
Sorry, Git highlighted something else in you review.

Reply:
This is correct now, thanks.